### ISSUE TITLE
Exception thrown 'java.lang.NoSuchMethodError' on calling notificationService.Show(notification);

### ISSUE THREAD

**Issue by LeoJHarris** [tmt242001/Plugin.LocalNotification#8]
Hi there using your plugin with XF 3.4 and target android API level 28, minimum API level 21.

I am getting the following exception:

referenceTable head length=54 111-30 18:09:52.458 F/art     ( 5985): art/runtime/check_jni.cc:65] JNI DETECTED ERROR IN APPLICATION: JNI CallObjectMethodA called with pending exception 'java.lang.NoSuchMethodError' thrown in void mono.java.lang.RunnableImplementor.n_run():-2

I am calling this:

ILocalNotificationService notificationService = DependencyService.Get<ILocalNotificationService>();
                                    LocalNotification notification = new Plugin.LocalNotification.LocalNotification
                                    {
                                        NotificationId =1,
                                        Title = "Title",
                                        Description = "Description",
                                        ReturningData = 1.ToString(), // Returning data when tapped on notification.
                                        NotifyTime = DateTime.Now.AddSeconds(2) // Used for Scheduling local notification.
                                    };
                                    **notificationService.Show(notification);**

Exception thrown on the code in bold. Not sure what the issue is, pluggin installed in both Forms and android project...

**Comment by tmt242001** [comment]
Please attach a sample solution

**Comment by masonyc** [comment]
I think your exception starts at here ILocalNotificationService notificationService = DependencyService.Get(); . Change it to ILocalNotificationService notificationService = DependencyService.Get<ILocalNotificationService>();

**Comment by LeoJHarris** [comment]
@tmt242001 Hi sorry for the delay, regarding the exception above I was able to resolve that particular issue. However I am currently getting an the following exception on the same line:

Java.Lang.NoSuchMethodError: no non-static method "Landroid/content/ContextWrapper;.checkSelfPermission(Ljava/lang/String;)I"

Now the issue is only happening on Android 5.1 (tested on both a tablet device and phone running 5.1), it could be happening on other versions as well, the plugin had no issue on android 7.1 though which I tested as well. It may just be affecting android lollipop but I haven't tested on others besides those described.

Seems to be API specific to android versions.
